README.md

```diff
@@ -34,4 +34,47 @@ For deploying documentation changes to the web, [Section 3 of rosdoc_lite wiki](
 
 ## Contributing
```

Comment:
Add in intro sentence for the contributing section encouraging everyone to make the tutorials better :)

Reply:
@davetcoleman done

Reply:
@davetcoleman done